recaptcha_enterprise/demosite/src/main/java/app/MainController.java

```diff
@@ -107,20 +107,15 @@ public static ModelAndView home() {
       // <!-- ATTENTION: reCAPTCHA Example (Server Part 1/2) Ends -->
 
       // Return the risk score.
-      HashMap<String, String> result = new HashMap<>() {{
-        put("score", String.valueOf(assessmentResponse.getRiskAnalysis().getScore()));
-        put("verdict", verdict);
-      }};
-      data = new HashMap<>() {{
-        put("data", result);
-      }};
+      HashMap<String, String> result = new HashMap<>();
+      result.put("score", String.valueOf(assessmentResponse.getRiskAnalysis().getScore()));
+      result.put("verdict", verdict);
+      data.put("data", result);
+
       return new ResponseEntity<>(data, httpHeaders, HttpStatus.OK);
     } catch (Exception e) {
-      data = new HashMap<>() {{
-        put("data", new HashMap<>() {{
-          put("error_msg", e.toString());
-        }});
-      }};
+      data.computeIfAbsent("data",
+          x -> new HashMap<>()).put("error_msg", e.toString());
```

Comment:
Sorry for being nitty. To make it clear (since this is a sample) can we assingn the return and do the put afterwards please?

```
Map<> dataMap = data.computeIfAbsent("data", x -> new HashMap<>());
dataMap.put("error_msg", e.toString());
```

Reply:
Yes, we could do that. Pushed a commit for it. PTAL